server/paths/api.ts

```diff
@@ -1,5 +1,32 @@
-import managePaths from './manage'
-import applyPaths from './apply'
+import { path } from 'static-path'
```

Comment:
I think this is a really good idea - combining application and API paths always make me a bit uneasy